packages/neos-ui-editors/src/Editors/Image/index.js

```diff
@@ -72,7 +72,9 @@ export default class ImageEditor extends Component {
                 isAssetLoading: true
             }, () => {
                 this.loadImage = loadImageMetadata(this.props.value.__identity).then(image => {
-                    this.setState({image, isAssetLoading: false});
+                    if (this._isMounted) {
```

Comment:
this probably fixes the flacky cropping tests? I also experienced issues here ^^